site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3452,7 +3452,11 @@ public function getSeekMessageByUserId($g_id, $user_id) {
             [$g_id, $user_id]
         );
 
-        return $this->course_db->rows()[0]['message'];
+        $rows = $this->course_db->rows();
+        if (empty($rows)) {
+            return null;
+        }
+        return $rows[0]['message'];
```

Comment:
```suggestion
        $row = $this->course_db->row();
        if (empty($rows)) {
            return null;
        }
        return $row['message'];
```

If we expect there to be one row all the time, we can use `row()` instead of `rows()[0]`.